.gitignore

```diff
@@ -10,6 +10,7 @@ __pycache__
 tests/acceptance/blockchain/.build/
 __local__.json
 /.cache
+.ruff_cache
```

Comment:
Technically, without the slash at the end, this ignores *any* file or directory called `.ruff_cache` and does work.
If the goal is just the directory in the root directory and its contents, we can use `.ruff_cache/` (similar to what we do for the `.pytest_cache` folder (L20).

Reply:
Right! Done ✅